s3/pkg/service/bucket.go

```diff
@@ -76,12 +77,32 @@ func (s *s3Service) CreateBucket(ctx context.Context, in *pb.Bucket, out *pb.Bas
 		return nil
 	}
 
+	// We need to find the backend in which bucket will be created
+	backend, err := utils.GetBackend(ctx, s.backendClient, in.DefaultLocation)
+	if err != nil {
+		log.Errorln("failed to get backend client with err:", err)
+		return err
+	}
+
+	sd, err := driver.CreateStorageDriver(backend.Type, backend)
```

Comment:
The create bucket API call will see error as "failed to create storage driver", Right? If yes, this is not very intuitive to the client

Reply:
It will return series of error message.
may be 1. failed to get backend client with err
             2. FRom where createbucket() called.